clustering/web/infinispan/src/main/java/org/wildfly/clustering/web/infinispan/session/fine/FineImmutableSessionAttributes.java

```diff
@@ -55,10 +57,19 @@ public FineImmutableSessionAttributes(String id, Cache<SessionAttributeCacheKey,
     @Override
     public Object getAttribute(String name) {
         SessionAttributeCacheKey key = this.createKey(name);
-        return this.marshaller.read(this.cache.get(key));
+        return this.read(name, this.cache.get(key));
     }
 
     protected SessionAttributeCacheKey createKey(String attribute) {
         return new SessionAttributeCacheKey(this.id, attribute);
     }
+
+    protected Object read(String name, V value) {
+        try {
+            return this.marshaller.read(value);
+        } catch (InvalidSerializedFormException e) {
+            InfinispanWebLogger.ROOT_LOGGER.failedToActivateSessionAttribute(e, this.id, name);
```

Comment:
Why is this logging and then re-throwing? This is generally considered to be an anti pattern, as the exception will usually be logged again at a higher level.